benchmarks/nx-cugraph/pytest-based/bench_algos.py

```diff
@@ -281,7 +281,7 @@ def bench_from_networkx(benchmark, graph_obj):
 @pytest.mark.parametrize("k", k_param_values, ids=lambda k: f"{k=}")
 def bench_betweenness_centrality(benchmark, graph_obj, backend_wrapper, normalized, k):
     G = get_graph_obj_for_benchmark(graph_obj, backend_wrapper)
-
+    print(k)
```

Comment:
We might want to clean up stdout and remove this.  I believe the `k` value is already part of the test name too?

Reply:
Ah, sorry about that.